frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -41,9 +191,59 @@ export function ElectionStatusPage() {
           </Button>
         </Alert>
       )}
-      <main>
-        <ElectionStatusProgress />
-        <article>Placeholder</article>
+      <main className={cls.statusMain}>
+        <div className={cls.statusTitle}>
+          <h2 id="status-title">{t("election_status.main_title")}</h2>
+          {/* TODO: Add button onClick to Create Polling Station page */}
+          <Button size="md" variant="secondary" leftIcon={<IconPlus />}>
+            {t("election_status.add_polling_station")}
+          </Button>
+        </div>
+        <div className={cls.statusSection}>
+          <Progress>
+            <div id="shortcuts" className="column">
+              <h2 className="mb-0">{t("shortcuts")}</h2>
```

Comment:
This section is has the heading shortcuts/snelkoppelingen, but they're not shortcuts.

Reply:
@jorisleker Shall I already add the onClick for the shortcuts to scroll to the specific table (or hide all other tables?) or add this to a separate issue for later on?

Reply:
Good catch @jschuurk-kr. This label indeed is a bit confusing. I'am not sure we need (anchor) links here, so best solution might be to update the heading. 

@Lionqueen94 can you update 'Snelkoppelingen' to 'Stembureaus per status'?

Reply:
Changed in 12c066a4625030ceb263d15c26a38d6c7d58875e